### ISSUE TITLE
Уведомление о загрузке большого количества графиков

### ISSUE THREAD

**Issue by bizlis** [SBProX/SBProX-Dev#1921]
Предлагаю установить галочку на данном окне и в общих настройках - "Больше не спрашивать"
![Screenshot_8](https://user-images.githubusercontent.com/29704362/111585718-8abd8680-87e1-11eb-9e92-d129bff2c3c0.jpg)

**Comment by WebSurfer001** [comment]
Поддерживаю

**Comment by oleg1ci** [comment]
Да, слишком много окошек на старте. Достаточно один раз настроить, зачем каждый день все эти уведомления читать?

**Comment by aleksandr-exclusive** [comment]
Поддерживаю

**Comment by ikki1561** [comment]
Поддерживаю.

**Comment by d13ded** [comment]
поддерживаю

**Comment by oleg1ci** [comment]
По идее надо убрать ВСЕ окна, в том числе окно выбора коннекторов,

**Comment by Vponed** [comment]
+
